app/static/init.uv.js

```diff
@@ -91,4 +91,10 @@ document.addEventListener("DOMContentLoaded", function () {
       uvElement.style.padding = "0.25rem";
     }
   }
+
+  document.querySelectorAll(".btn").forEach((button) => {
```

Comment:
so this is an example of us fixing acc issues that ideally would be done by uv themselves. Going forward like we discussed I think we will hold off doing more of this and have a conversation with UV/ try make prs in their repo